### PR TITLE
update master branch latest snapshot version

### PR DESCRIPTION
**Pull request by sophiaso**
## Summary
- Update master branch snapshot version
- Manage json version in parent pom

## Issue Type
- Bug fix

## Starter Names
  - active directory spring boot starter
  - documentdb spring boot starter
  - key vault spring boot starter
  - media services spring boot starter
  - service bus spring boot starter
  - storage spring boot starter

## Additional Information

### CODE REVIEW

File: azure-spring-boot-bom/pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
     <groupId>com.microsoft.azure</groupId>
     <artifactId>azure-spring-boot-bom</artifactId>
-    <version>0.2.4-SNAPSHOT</version>
+    <version>2.0.1-SNAPSHOT</version>
```
Comment (yungez) [inline azure-spring-boot-bom/pom.xml]: version in readme files need to be updated also.

Comment (sophiaso) [reply azure-spring-boot-bom/pom.xml]: @yungez Updated.

File: azure-spring-boot-parent/pom.xml [hunk 2]
```diff
@@ -75,6 +75,11 @@
                 <artifactId>powermock-api-mockito2</artifactId>
                 <version>1.7.0</version>
             </dependency>
+            <dependency>
+                <groupId>org.json</groupId>
+                <artifactId>json</artifactId>
+                <version>20171018</version>
+            </dependency>
```
Comment (sophiaso) [inline azure-spring-boot-parent/pom.xml]: refs #283

**Comment by yungez** [comment]
@snicoll

**Pull request merged**